src/05-invoicing/invoices.apib

```diff
@@ -184,6 +184,13 @@ Draft a new invoice.
                                     + excluding
                         + tax_rate_id: `c0c03f1e-77e3-402c-a713-30ea1c585823` (string, required)
         + invoice_date: `2016-02-04` (string, optional)
+        + discounts (array)
+            (object)
+                + value: 0.5123 (number, required)
+                + type (enum[string], required)
+                    + Members
+                        + percentage
+                + description: `winter promotion` (string) - If the description is not given, the proper translation of the word discount will be used in the user interface
```

Comment:
Let's do it in multiple phases and leave that comment out for now (or make the string 'optional', but I think it's better to take smaller steps)